components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
in order to aviod linting errors with `is-clickable' class name, you should insert this line at the top of the document

`/* sass-lint:disable no-important, class-name-format*/`